app/src/main/java/com/x8bit/bitwarden/data/autofill/util/AutofillValueExtensions.kt

```diff
@@ -26,7 +26,11 @@ fun AutofillValue.extractMonthValue(
 /**
  * Extract a text value from this [AutofillValue].
  */
-fun AutofillValue.extractTextValue(): String? = this
-    .textValue
-    .takeIf { it.isNotBlank() }
-    ?.toString()
+fun AutofillValue.extractTextValue(): String? =
+    if (isText) {
+        textValue
+            .takeIf { it.isNotBlank() }
```

Comment:
I'm curious about the formatting of having one condition in the `if (...)` and one condition in the `.takeIf`. Why not combine them like `.takeIf { it.isNotBlank() && isText }`?

Reply:
The `if` is required since the crash occurs as soon as you call `textValue`, so we have to cal `isText` first.

Reply:
Yeah I was wondering the same. Looks like it would blow up before that point unfortunately.

Reply:
Got it, thanks!